packages/plugin-e2e/tests/datasource/variables/customVariableEditor.spec.ts

```diff
@@ -20,7 +20,7 @@ test('should load resources and display them as options when clicking on an inpu
   await variableEditPage.datasource.set(provision.datasources?.[0]!.name!);
   await page.getByLabel('Schema').click();
   await expect(variableEditPage.getByTestIdOrAriaLabel('Select option')).toContainText(REDSHIFT_SCHEMAS);
-  await page.keyboard.press('Escape');
+  await page.keyboard.press('Enter');
```

Comment:
Is this change intended?

Reply:
Yes - In Grafana 9.2.x (and earlier), hitting Escape takes you away from dashboard settings and back to the dashboard start page. So needed some other command that can move focus away from the select, but still stay on the same page.